src/views/Pools/components/LockedPool/Modals/ExtendDurationModal.tsx

```diff
@@ -77,6 +91,7 @@ const ExtendDurationModal: React.FC<ExtendDurationModal> = ({
         </Box>
         <LockedBodyModal
           stakingToken={stakingToken}
+          hasEnoughBalanceToExtend={currentBalance?.gt(MIN_AMOUNT)}
           onDismiss={onDismiss}
           lockedAmount={new BigNumber(currentLockedAmount)}
```

Comment:
could you help me add `useMemo` for this prop to avoid re-render?

Reply:
Done